Makefile

```diff
@@ -46,8 +48,18 @@ travis: test-verbose lint
 unit:
 	go test $(GO_TEST_FLAGS) ./cmd/... ./pkg/...
 
+.PHONY: unit-cover
+unit-cover:
+	go test -coverprofile=cover.out $(GO_TEST_FLAGS) ./cmd/... ./pkg/...
+	go tool cover -func=cover.out -o coverage-stats.out
```

Comment:
Ideally we would have a target that reports the stats to the console, so we can run it in CI and see the results in the logs. Is there a separate mode for doing that, or do we just need to cat the output file here?

Reply:
If there is no output file defined, the stats will be reported directly to he console. I'll do that.